cocoa/neural/evaluator.py

```diff
@@ -82,7 +82,7 @@ def evaluate(self, opt, model_opt, data, split='test'):
                 gold_words_total += len(response.gold_sent)
 
             if opt.verbose:
-                self.print_results(model_opt, batch, utterances)
+                counter = self.print_results(model_opt, batch, counter, utterances)
```

Comment:
what's the counter for?

Reply:
the counter is for knowing the example number, and for extracting the response (which depends on the sentence number).  originally, when the evaluate() and print_results() were one function the counter did not need to be passed around, but now it does.  we could create some more global counter, but that seems messy as well